### DISCUSSION TITLE
Example file from the inverter

### DISCUSSION THREAD

**Discussion by Suzukithunder** [johanmeijer/grott#482] [General]
Hi, does any of you have an encrypted and unencrypted example of the traffic sent from the inverter to the server? So I can see tag names, values etc. Trying to get up to speed on this project.

**Comment by johanmeijer** [comment]
Isn't that the whole idea of this software? Run grott (with the right parameters defined for your inverter) and you will see this infromation in the grott log.

**Reply by Suzukithunder** [reply]
Hi Johan. Perhaps I did something wrong. I get errno 98, address already in use. Any guidance ? :_)

**Reply by Suzukithunder** [reply]
I figures it out. Now struggling with csv file :SD

**Comment by johanmeijer** [comment]
Do you use the grottcsv extention?

https://github.com/johanmeijer/grott/wiki/Extensions

**Reply by Suzukithunder** [reply]
Hi Johan,

I have gotten a lot further. I see numbers in the Terminal and it is rerouted correctly. Only issue is that I dont see any data in influxdb2 at all. Ip and ports are all okay. Any suggestions?

**Comment by Suzukithunder** [comment]
Hi Johan,

 

Yes I try to. No file created for yesterday, but I see a lot of traffic in the terminal. Trying to troubleshoot.

 

Med venlig hilsen / Bästa hälsningar / Best Regards / Mit freundlichen Grüssen

Niels Hvid
NH Handel ApS
Cirkelhusene 10
4100 Ringsted
Denmark 
CVR Nr.: 32344399
Mobil: 0045 31 51 95 66
Email:  ***@***.***> ***@***.***
Web: www.nhhandel.dk <http://www.nhhandel.dk> 

Denne e-mail og enhver vedhæftet fil er fortrolig og alene tiltænkt modtageren. Såfremt De ikke er rette modtager, vil adgang til e-mailen være ubeføjet, og enhver udnyttelse eller videregivelse af e-mailens indhold være uretmæssig og potentielt i strid med lovgivningen. Såfremt De ikke er rette modtager, bedes De venligst omgående underrette os og derefter slette e-mailen og enhver vedhæftet fil. På forhånd tak.

The information in this email and any attachment is confidential and may be legally protected. It is intended solely for the addressee. Access to this email by anyone else is unauthorised. If you are not the intended recipient, any disclosure or actions taken as a result of the information in this email is prohibited and may be unlawful. If you are not the intended recipient, please notify us immediately and then delete this e-mail and any attachment. Thank you.

 

Fra: Johan Meijer ***@***.***> 
Sendt: 24. december 2023 13:42
Til: johanmeijer/grott ***@***.***>
Cc: Suzukithunder ***@***.***>; Author ***@***.***>
Emne: Re: [johanmeijer/grott] Example file from the inverter (Discussion #482)

 

Do you use the grottcsv extention?
